### PR TITLE
Update pymapd

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* ~[ ] Reset the build number to `0` (if the version changed)~
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xmnlab** [comment]
@conda-forge/ibis-framework in this PR:

- it is using noarch: python
- it uses python >=3
- dependencies for py2 and py35 were removed
- it uses the latest packages for pymapd, pyarrow and impyla
- included me (xmnlab) as maintainer

**Comment by xmnlab** [comment]
@kszucs I am adding you as maintainer :+1:

**Comment by xmnlab** [comment]
hey everyone, any thoughts about this?

**Comment by wesm** [comment]
Looks OK to me. Will defer to @kszucs

**Comment by xmnlab** [comment]
thanks for the feedback @wesm ! @kszucs any thoughts about this PR?

File: recipe/meta.yaml
Comment (kszucs) [inline recipe/meta.yaml:14]: Nice!

**Comment by kszucs** [comment]
The new build would ship new features and updates the dependencies, so I think We should bump the version too, thus draft an ibis release. What do you think @cpcloud ?

**Comment by cpcloud** [comment]
@kszucs SGTM. Can we get https://github.com/ibis-project/ibis/pull/1714 in before we cut the release?

File: .azure-pipelines/azure-pipelines-osx.yml
Comment (cpcloud) [inline .azure-pipelines/azure-pipelines-osx.yml:14]: Why are we building for 2.7?

File: .azure-pipelines/azure-pipelines-win.yml
Comment (cpcloud) [inline .azure-pipelines/azure-pipelines-win.yml:14]: Same question as above: why 2.7?

**Review by cpcloud — CHANGES_REQUESTED** [review]

**Comment by cpcloud** [comment]
It looks like this didn't build on Azure, is that correct?

**Comment by kszucs** [comment]
@cpcloud sure, as soon as the build turns green

**Comment by kszucs** [comment]
@conda-forge-admin please rerender

File: .azure-pipelines/azure-pipelines-osx.yml
Comment (kszucs) [reply .azure-pipelines/azure-pipelines-osx.yml:14]: The azure builds are not running at all which I don't understand why are they defined at all.

It might be because We're not using selectors, just constraints for `py>3`. I can't find the conda issue, but the noarch py3k packages are supposed to be defined like @xmnlab did.

**Comment by cpcloud** [comment]
Should we merge this? @xmnlab any idea what's up with the 2.7 showing up?

**Comment by kszucs** [comment]
We should cut a release first, otherwise new api and dependencies are going to be installed without a version bump.

**Comment by xmnlab** [comment]
hey @cpcloud ! I think py27 shows up because, as @kszucs said, this recipe doesn't use build skip with selector ... it just has a constraint for py>=3 .. I think it shows up but the packaging will not be triggered.

**Comment by xmnlab** [comment]
also azure ci is used by conda forge but it is still in test. see https://github.com/conda-forge/double-conversion-feedstock/pull/9#issuecomment-471276790

**Comment by xmnlab** [comment]
@kszucs what is the plan for the next release?

**Comment by cpcloud** [comment]
I will go through the issue log this week and move/close/PR things accordingly. Let's shoot for Monday as the release target.

**Comment by xmnlab** [comment]
@cpcloud 

awesome! I am working on https://github.com/ibis-project/ibis/pull/1710
I think it is done for a review (3 jobs is still pending on circleci, but the latest commits was about the azure) .. as I mentioned in this PR, the conda task is deprecated, so now it is necessary to create environment manually and call `conda activate` inside each `script` section.

if you need any help let me know.

**Comment by cpcloud** [comment]
Can we test this out using a hash without cutting a release?

**Comment by cpcloud** [comment]
Alternatively, I'm okay with mutating the 1.0.0 tag if we find something that needs to be fixed on the ibis side before the official release announcement is made.

**Comment by xmnlab** [comment]
@cpcloud I can change to use the hash for test (I will do it tomorrow)

**Comment by cpcloud** [comment]
@xmnlab This is ready to try with a77308c981a75e9c24330d2a2766fa65c0d1d6e5

**Comment by xmnlab** [comment]
@cpcloud done! I changed it. it seems it is working.

**Comment by xmnlab** [comment]
I will change now the pinning for pymapd to >=0.8.3 and check if it is working

**Comment by cpcloud** [comment]
Sounds good. When that passes I will tag upstream.

**Comment by xmnlab** [comment]
all green :+1: 
when the tag is done I can change again this recipe.

**Comment by xmnlab** [comment]
@cpcloud you create a tag without `v` (just 1.0.0).
is it correct? if yes I can remove `v` from the url

**Comment by cpcloud** [comment]
That's right

**Comment by xmnlab** [comment]
done!

**Comment by cpcloud** [comment]
@xmnlab can you repush again here? I had to fix up the versioneer script to remove the tag prefix `v` in ibis master.

**Review by cpcloud — APPROVED** [review]

**Comment by xmnlab** [comment]
sure! it is done! :)

**Comment by cpcloud** [comment]
LGTM! Merging.

**Pull request merged**
